webapi/appsettings.json

```diff
@@ -38,8 +38,8 @@
   //
   "AIService": {
     "Type": "AzureOpenAI",
-    "Endpoint": "https://lightspeed-team-shared-openai.openai.azure.com/", // ignored when AIService is "OpenAI"
-    "Key": "9e2fe86fbbfd4438bdc9bf81e8a1a9c3",
+    "Endpoint": "", // ignored when AIService is "OpenAI"
+    "Key": "",
```

Comment:
There can still be risk doing it this way because the creds will remain in the prior commit.  Can you revert the commit that originally added your creds instead of removing creds in additional commit? (I've been in this same boat before too)

Reply:
Too late. We'll generate new keys instead.